templates/docker.service.j2

```diff
@@ -4,14 +4,13 @@
 Description=Docker Application Container Engine
 Documentation=https://docs.docker.com
 After=network.target docker.socket firewalld.service
-Requires=docker.socket
 
 [Service]
 Type=notify
 # the default is not to use systemd for cgroups because the delegate issues still
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
-ExecStart=/usr/bin/dockerd -H fd:// {{ '-H ' + docker_remote_socket if docker_remote_socket != '' else '' }} {{ '-H ' + docker_local_socket if docker_local_socket != '' else '' }}
+ExecStart=/usr/bin/dockerd {{ '-H ' + docker_remote_socket if docker_remote_socket != '' else '' }} {{ '-H ' + docker_local_socket if docker_local_socket != '' else '' }}
```

Comment:
Why this is changed?

Reply:
When I removed line "Requires=docker.socket" I had to also removed this option. Without it I could not install Docker.